README.md

```diff
@@ -0,0 +1,203 @@
+
+  <h3><b>trials-review README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [trials-Review] <a name="about-project"></a>
+
+
+**[trials-review]** is a test project for my microverse review study.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>JAVASCRIPT</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Contains an index file]**
+- **[Has a test suite]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- The project is not deployed yet.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+git installed.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd trials-review
+  git clone https://github.com/Obimbo07/trials-review.git
+```
+
+### Install
+
+Install this project with:
+
+git 
+Terminal
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```
+   cd trials-review
+   open index.html
+
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+execute babel file
+```
+
+### Deployment
+
+You can deploy this project using:
+
+github pages.
+
```

Comment:
Add a deployment link to your project so as to make the readme more professional